api/base.py

```diff
@@ -241,9 +241,10 @@ def set_origin(self, drone_request):
                 }, {
                     '$set': {
                         '_id': self.origin['id'],
-                        'last-seen': datetime.datetime.utcnow(),
+                        'last_seen': datetime.datetime.utcnow(),
                         'method': self.origin['method'],
-                        'name': self.origin['name']
+                        'name': self.origin['name'],
+                        'errors': None # Reset errors list if device checks in
```

Comment:
Can we please set `errors` to an empty array instead of null? Consistent typing.